src/wren/GlState.cpp

```diff
@@ -120,15 +120,18 @@ namespace wren {
       cVersion = reinterpret_cast<const char *>(glGetString(GL_VERSION));
       cGlslVersion = reinterpret_cast<const char *>(glGetString(GL_SHADING_LANGUAGE_VERSION));
 
-      if (GLAD_GL_ATI_meminfo)
-        glGetIntegerv(GL_TEXTURE_FREE_MEMORY_ATI, &cGpuMemory);
-      else if (GLAD_GL_NVX_gpu_memory_info)
+      if (GLAD_GL_NVX_gpu_memory_info)
         glGetIntegerv(GL_GPU_MEMORY_INFO_TOTAL_AVAILABLE_MEMORY_NVX, &cGpuMemory);
-      else
-        DEBUG("GL_TEXTURE_FREE_MEMORY_ATI and GL_GPU_MEMORY_INFO_TOTAL_AVAILABLE_MEMORY_NVX extensions not supported by "
-              "hardware, impossible to detect GPU memory"
-              << std::endl);
-
+      else {
+        // Try to use GL_TEXTURE_FREE_MEMORY_ATI:
+        // it seems to be working even if the corresponding GLAD_GL_ATI_meminfo is not available
+        GLint array[4];
+        array[0] = -1;
+        glGetIntegerv(GL_TEXTURE_FREE_MEMORY_ATI, array);
```

Comment:
We should probably call: https://github.com/omichel/webots/blob/c99c027abbb4316d52abe46980398ca4c568772a/src/wren/GlState.cpp#L732
Just before this to make sure that we don't skip other error while skipping the error generated by this enum.